manubot/process/head-template.html

```diff
@@ -0,0 +1,79 @@
+<!--
+https://github.com/jgm/pandoc-templates/blob/master/default.html5
+https://github.com/manubot/rootstock/issues/176
+https://github.com/manubot/rootstock/issues/203
+https://scholar.google.com/intl/en/scholar/inclusion.html#indexing
+https://larsgw.blogspot.com/2018/07/journal-metadata-authors-institutions.html
+https://ogp.me
+https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup.html
+-->
+
+<!-- Manubot generated metadata -->
+{% if title is defined %}
+<meta name="citation_title" content="{{ title|escape }}">
+<meta name="DC.title" content="{{ title|escape }}">
+<meta property="og:title" content="{{ title|escape }}">
+<meta property="twitter:title" content="{{ title|escape }}">
+{% endif %}
+{% if iso_date is defined %}
+<meta name="citation_publication_date" content="{{ iso_date }}">
+{% endif %}
+{% if abstract is defined %}
+<meta name="DC.title" content="{{ abstract|escape }}">
```

Comment:
Incorrect?

Reply:
Or perhaps should set `citation_abstract` or `DC.Description` like bioRxiv and not `DC.abstract`.